src/ssmDocument/commands/createDocumentFromTemplate.ts

```diff
@@ -34,7 +34,20 @@ const SSMDOCUMENT_TEMPLATES: SsmDocumentTemplateQuickPickItem[] = [
         docType: 'command',
     },
     {
-        label: localize('AWS.ssmDocument.template.automationJson.label', 'JSON Automation Document 0.3'),
+        label: localize('AWS.ssmDocument.template.command22Yaml.label', 'Command Document (schemaVersion 2.2, YAML)'),
```

Comment:
Could you move the Automation entries above the Command entries? Looking at the screenshot, its reasonable to think these should be in alphabetical order. Unless SSM Docs have a process flow where Command docs precede Automation docs...